usaspending_api/agency/v2/views/budgetary_resources.py

```diff
@@ -18,82 +16,58 @@ class BudgetaryResources(AgencyBase):
+                agency_total_obligated=Sum("obligations_incurred_total_by_tas_cpe"),
+            )
         )
-        return aab["agency_budgetary_resources"], aab["agency_total_obligated"]
 
-    def get_prior_year_agency_budgetary_resources(self):
-        """
-        Even though we're looking at fiscal_year - 1, it's possible that fiscal year hasn't been closed out
-        yet.  For example, if today is 5 Oct 1999 then the submission window for FY 1999 is not closed yet
-        even though we're in FY2000 Q1.
-        """
-        prior_fiscal_year = self.fiscal_year - 1
-        prior_fiscal_year_last_completed_fiscal_period = get_final_period_of_quarter(
-            calculate_last_completed_fiscal_quarter(prior_fiscal_year)
-        )
-        return AppropriationAccountBalances.objects.filter(
-            submission__reporting_fiscal_year=prior_fiscal_year,
-            submission__reporting_fiscal_period=prior_fiscal_year_last_completed_fiscal_period,
-            treasury_account_identifier__funding_toptier_agency=self.toptier_agency,
-        ).aggregate(agency_budgetary_resources=Sum("total_budgetary_resources_amount_cpe"))[
-            "agency_budgetary_resources"
-        ]
-
-    def get_agency_obligations_by_period(self):
-        """
-        We limit periods to 3, 6, 9, and 12 (quarters) but with the CARES Act coming down the road
-        we will need to figure out how to include monthly submissions.  I am kicking that bucket down
-        the road until the CARE Act details have been more fully fleshed out.
-        """
-        fiscal_periods = [n for n in (3, 6, 9, 12) if n <= self.fiscal_period]
-        return [
+        results = [
             {
-                "period": abb["submission__reporting_fiscal_period"],
-                "obligated": abb["obligations_incurred_total_by_tas_cpe__sum"],
+                "fiscal_year": x["submission__reporting_fiscal_year"],
+                "agency_budgetary_resources": x["agency_budgetary_resources"],
+                "agency_total_obligated": x["agency_total_obligated"],
+                "federal_budgetary_resources": self.get_total_federal_budgetary_resources(
+                    x["submission__reporting_fiscal_year"]
+                ),
             }
-            for abb in (
-                AppropriationAccountBalances.objects.filter(
-                    submission__reporting_fiscal_year=self.fiscal_year,
-                    submission__reporting_fiscal_period__in=fiscal_periods,
-                    treasury_account_identifier__funding_toptier_agency=self.toptier_agency,
-                )
-                .values("submission__reporting_fiscal_period")
-                .annotate(Sum("obligations_incurred_total_by_tas_cpe"))
-                .order_by("submission__reporting_fiscal_period")
-            )
+            for x in aab
         ]
+        years = [x["fiscal_year"] for x in results]
+        for year in range(2017, current_fiscal_year() + 1):
```

Comment:
Need to ensure we thought of edgecases with the Frontend. If `submission_reveal_date` is used, it means the next FY won't contain data until after P2 submissions are due. (I believe this would be early January) So from Oct 1st - January X the fiscal year would be provided to the frontend but would be a blank result. It makes sense to me but might be confusing to api users.